gsa/src/web/pages/alerts/method.js

```diff
@@ -59,13 +62,26 @@ const Table = styled(SimpleTable)`
   }
 `;
 
-const Method = ({method = {}, details = false}) => {
+const Pre = styled.pre`
+  white-space: pre-wrap;
+  word-wrap: break-word;
+`;
+
+const Method = ({method = {}, details = false, reportFormats}) => {
   if (!isDefined(method.type)) {
     return null;
   }
+
+  const getReportFormatName = id => {
+    if (isDefined(reportFormats)) {
+      return reportFormats.find(format => format.id === id).name;
```

Comment:
This may crash if the report format couldn't be found. Therefore it should be changed to
```suggestion
      reportFormat = reportFormats.find(format => format.id === id);
      if (isDefined(reportFormat)) {
        return reportFormat.name;
      }
      return null;
```